### PR TITLE
fix(cli): panic on empty commit or empty type

### PR DESCRIPTION
**Pull request by suyulin**
# Why
commit message is "", it's panic。
commit message invalid type ,it's panic.  like this “xx”。

### CODE REVIEW

**Comment by KeisukeYamashita** [comment]
Thank you, I'll take a look tonight!

File: src/rule/type.rs [hunk 1]
```diff
@@ -122,4 +122,42 @@ mod tests {
...
+            subject: None,
+        };
+        assert_eq!(rule.validate(&message).unwrap().level, Level::Error);
+        assert_eq!(
+            rule.validate(&message).unwrap().message,
+            "type  is not allowed. Only [] are allowed".to_string());
+    }
+
+    #[test]
+    fn test_invalid_raw() {
```
Comment (KeisukeYamashita) [inline src/rule/type.rs:146]:
```suggestion
    fn test_missing_type() {
```

**Review by KeisukeYamashita — CHANGES_REQUESTED** [review]
Thank you very much, PTAL 🙏

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 /target
+/.idea
```
[the remark this answers is not in the document]
Comment (suyulin) [reply .gitignore]: thx, I fixed it

**Review by KeisukeYamashita — APPROVED** [review]
I've updated the PR name to match or format, thank you for your contribution.
It will be shipped soon 🏃

**Pull request merged**